docs/custom-pools.md

```diff
@@ -4,6 +4,8 @@ Custom pools are pools that inherit from the `worker` pool. They use any Machine
 but add the ability to deploy changes only targeted at the custom pool.
 Since a custom pool inherits from the `worker` pool, any change to the `worker` pool is going to roll out to the custom pool as well (like an OS update during an upgrade).
 
+**Note:** Custom pools on master nodes are not supported by MCO. For more detail, see [Custom pool on master node](#Custom-pool-on-master-node)
```

Comment:
if you pull this up to view the file normally the link seems to be missing - might be the capitalization? try (#custom-pool-on-master-node)